tests/xml.js

```diff
@@ -150,6 +152,18 @@ define([
             eq('<output attr=value />', '<output attr="value" />', false);
         });
 
+        it("should fail on escape unquoted attribute value with <", function () {
+            chai.expect(function () {
+                eq('<output value=2<3 />', '<output value="2&lt;3" />', false);
```

Comment:
The result of `xml.normalize('<output value=2<3 />')` on phantomjs is

`<output value="2&lt;3"><!--3--></output>`

which is very strange indeed. It's even semantically equivalent XML, which is nice.